go/client/cmd_team_list_memberships.go

```diff
@@ -73,6 +74,10 @@ func newCmdTeamListMemberships(cl *libcmdline.CommandLine, g *libkb.GlobalContex
 				Usage: "Include any subteam memberships as well",
 			},
 			cli.BoolFlag{
+				Name:  "include-implicit-teams",
```

Comment:
I don't think users ever should know about implicit teams. The rpc flag could still be useful if we want a dev-only command for listing implicit teams.